lib/features/modeling/cmd/LinkCallChoreoHandler.js

```diff
@@ -0,0 +1,51 @@
+/**
+ *
+ */
+export default function LinkCallChoreoHandler() {}
+
+/**
+ * Links the a choreo to a call choreo and un-links the old one if necessary.
+ * @param element {businessObject}
+ * @param newLink {businessObject} undefined if element should not be linked
+ * @param oldLink {businessObject} undefined if element was not linked
+ * @returns {*[]}
+ */
+LinkCallChoreoHandler.prototype.link = function(element, newLink, oldLink) {
```

Comment:
I don't really see why you need to pass around `oldLink`. In my opinion, it should be possible to realize the whole command with just `callChoreo` and `newLink`:

- in `preExecute` set `context.oldLink = context.callChoreo.calledChoreographyRef`
- in `execute`, call `this.link(context.callChoreo, context.newLink)`
- in `revert`, call `this.link(context.callChoreo, context.oldLink)`
- in `link(callChoreo, updateLink)`:
  - unlink  `context.callChoreo.calledChoreographyRef`
  - set `context.callChoreo.calledChoreographyRef = updateLink`
  - add the call choreo to `updateLink`s `callChoreographyActivity` array

Am I missing something?